sql/rikai--0.1.sql

```diff
@@ -6,6 +6,14 @@ CREATE TYPE image AS (uri TEXT);
 
 CREATE TYPE detection AS (label TEXT, label_id int, box box, score real);
 
+-- rikai.types.Mask
+CREATE TYPE mask AS (
+    type INT,
```

Comment:
what's the type parameter again? whether it's RLE or polygon representation?

Reply:
https://github.com/eto-ai/rikai/blob/main/python/rikai/types/geometry.py#L550-L555 
it is used differentiate 2 RLE types and polygons